pkgs/build-support/lib/meson.nix

```diff
@@ -23,6 +23,11 @@ let
     [binaries]
     llvm-config = 'llvm-config-native'
     rust = ['rustc', '--target', '${stdenv.targetPlatform.rust.rustcTargetSpec}']
+    # Credits to Qyriad for finding the following:
```

Comment:
Feels a bit weird to credit a bug's discoverer in a comment to me.  If we did that consistently there'd be so much noise.

Reply:
Yeah, I should just co-author in the commit message.

Reply:
Done.